### PR TITLE
Commented out broken links in openfirmware docs causing bamboo build to fail

### PR DESCRIPTION
**Pull request by kylekirkby**
@shovanuk

### CODE REVIEW

File: openfirmware-docs/1275/practice/devicex/dse1_0a.html [hunk 1]
```diff
@@ -464,14 +464,20 @@
 <H2><A NAME="HDR22">8.   Serial port device</A></H2>
 Access to the serial port is supported by this device type.  The serial port is treated as a byte-stream device.<BR>
 <H3><A NAME="HDR23">8.1.   Serial port properties</A></H3>
-As specified in <A HREF="dse1_0a.html#REF37838">[1]</A> and <A HREF="dsev10a.html#REF33177">[6]</A> , with the following additions or modifications.<BR>
+As specified in <A HREF="dse1_0a.html#REF37838">[1]</A> and
+<!-- <A HREF="dsev10a.html#REF33177">[6]</A> -->
```
[the remark this answers is not in the document]
Comment (kylekirkby) [reply openfirmware-docs/1275/practice/devicex/dse1_0a.html]: Hi Rob,

We are using a link checker on our static site builds that stops the site from building if internal broken links are found. If possible, can you fix these links or accept them commented out for now so the build can go ahead.

Thanks,

Kyle

**Comment by shovanuk** [comment]
@robherring Let me know if you are happy for me to accept the PR

File: openfirmware-docs/1275/practice/devicex/dse1_0a.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (robherring) [reply openfirmware-docs/1275/practice/devicex/dse1_0a.html]: Okay, that's fine to leave them commented, but please squash the whitespace changes to the original commit.

[the remark this answers is not in the document]
Comment (kylekirkby) [reply openfirmware-docs/1275/practice/devicex/dse1_0a.html]: Removed all the whitespace changes - sorry about that.

Thanks,

Kyle

**Pull request merged**